### PR TITLE
Update recommended-settings-for-eop-and-office365-atp.md

### PR DESCRIPTION
**Pull request by andypunt**
Add tip to make sure Junk Email Rule is enabled.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365-atp.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ description: "What are best practices for Exchange Online Protection (EOP) and A
 
 Although we empower security administrators to customize their security settings, there are two security levels in EOP and Office 365 ATP that we recommend: **Standard** and **Strict**. Each customer's environment and needs are different, but we believe that these levels of mail filtering configurations will help prevent unwanted mail from reaching your employees' inbox in most situations.
 
+> [!IMPORTANT]
+> The Junk Email Configuration has to be enabled on the mailbox in order for filtering to work properly. This is enabled by default, but should be checked if filtering does not seem to be working. [Read Set-MailboxJunkEmailConfiguration to find out more](https://docs.microsoft.com/en-us/powershell/module/exchange/antispam-antimalware/set-mailboxjunkemailconfiguration).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365-atp.md]:
```suggestion
> The Junk Email Configuration has to be enabled on the mailbox in order for filtering to work properly. This is enabled by default, but should be checked if filtering does not seem to be working. Read [Set-MailboxJunkEmailConfiguration](https://docs.microsoft.com/powershell/module/exchange/antispam-antimalware/set-mailboxjunkemailconfiguration) to find out more. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@andypunt - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review.  


Thank you.

cc: @kenwith

**Pull request merged**